### ISSUE TITLE
msg time iterating over file is seconds, not ticks

### ISSUE THREAD

**Issue by nathan-stewart** [mido/mido#477]
1.2.10 docs say:
The time attribute is used in several different ways:

    inside a track, it is delta time in ticks. This must be an integer.
    in messages yielded from play(), it is delta time in seconds (time elapsed since the last yielded message)

```
import mido
f = mido.MidiFile('demo/cwm_rhondda.mid')
for m in f.play():
    print(m.time)

```
shows time in seconds as expected
```
...
0
0.4739583333333333
```

`f.tracks[0][13]    `
shows the time in ticks as expected:
`Message('note_on', channel=0, note=55, velocity=0, time=455)`

```
for m in f:
    print(m.time) shows the time in seconds, not ticks.
```

There doesn't appear to be a way to get all messages across all tracks in the proper time sequence as with play while getting the time in ticks. I believe this behavior is broken.

**Comment by rdoursenaud** [comment]
Yes, I believe that the `time` attribute should be revamped to not mix units, allow conversion to and from each and support both relative and absolute time.
This is something I have had in the back of my mind for quite a while but I haven't looked at it seriously.
Thanks for the reminder and, to anyone, feel free to discuss and suggest ways to improve this area.

**Comment by nathan-stewart** [comment]
I haven't dug into the code but it seems to me that iterating over the file should return the actual messages, and play should preserve its current behavior.  An alternative might be to mirror the time field with a ticks field but my head isn't clear on this right now. It might have the advantage of not breaking anything written against current behavior though. 

I don't know how people are using this. I myself am using  for msg in midifile.play(): when I actually play the notes, and trying to do operations building the midifile using for msg in midifile: 

 (I've got a web app MIDI metronome.  I recently switched from just sending midi messages directly to building a file so I don't have to rebuild my table whenever tempo  or subdivision note or volume changes. Only if  the time signature changes or the swing setting changes do I rebuild the midifile. But since I'm using a midifile now I added the ability to load midi files and play them at tempo. I found that measure/beats were fine but once I added eighth/sixteenths/clock messages things no longer matched the loaded file. I'm not yet sure if this is related to this bug/my handling of the files yet, or something else.)
